src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -34,6 +35,7 @@ public class AddCommand extends Command {
             + PREFIX_TO + "TO] "
             + "[" + PREFIX_TAG + "TAG]...\n"
             + "[" + PREFIX_MOD + "MOD]...\n"
```

Comment:
I think we can remove the ...\n at the end, and just leave a single whitespace after the closing square bracket.